apps/docs/pages/builderdao-cli/solana_airdrop.md

```diff
@@ -0,0 +1,106 @@
+# The airdop command
+
+The following guide shows you how to send $KAFE and $BDR tokens to alpha testnet users.
+
+<aside>
```

Comment:
Is the aside component tage here coming from Notion, or is it built in to the Nextra renderer? I thought we needed to use the `.mdx` file extension and import the React components we want to use. Please correct if I am wrong!